### DISCUSSION TITLE
Add Skip Icons

### DISCUSSION THREAD

**Discussion by phix-o** [tailwindlabs/heroicons#1001] [Ideas]
It would be nice if you could add the `skip forward` and `skip back` icons similar to those of media playback icons.  Thanks.

**Comment by lxrst** [comment]
There's already `forward`, `backward`, `arrow-uturn-left`, and `arrow-uturn-right`.

**Comment by kevinashworth** [comment]
I have been using the `forward` icon for a purpose I have been calling "skip" and after a while I came to the site to find something even better. I was surprised not to find a skip icon. So, for what it's worth, this is a +1 vote for skip icons like at https://github.com/tailwindlabs/heroicons/pull/998
